packages/aws-cdk/lib/init.ts

```diff
@@ -320,8 +329,8 @@ async function postInstallPython(cwd: string) {
   try {
     await execute(python, ['-m venv', '.env'], { cwd });
   } catch (e) {
-    print('Unable to create virtualenv automatically');
-    print(`Please run ${colors.green(python + ' -m venv .env')}!`);
+    warning('Unable to create virtualenv automatically');
+    warning(`Please run ${colors.green(python + ' -m venv .env')}!`);
```

Comment:
Drop the green part of this message, so it's all yellow.

Reply:
done